guides/ig-rezeptanforderung/Home/Datenobjekte/MedicationRequest/3_Profil.page.md

```diff
@@ -0,0 +1,18 @@
+## {{page-title}}
+
+Die folgende FHIR Ressource stellt eine Ressource bereit, die als Trägerobjekt für medizinische Informationen dient. Sie ist von der Profilierung an die [KBV_PR_ERP_Prescription](https://simplifier.net/erezept/kbvprerpprescription) angelehnt, um Informationen so einfach wie möglich in die Verordnung zu übernehmen.
```

Comment:
Sie ist an die [KBV_PR_ERP_Prescription](https://simplifier.net/erezept/kbvprerpprescription) Profilierung angelehnt, um Informationen so einfach wie möglich in die Verordnung zu übernehmen.